apis/kueue/v1alpha1/tas_types.go

```diff
@@ -27,3 +53,57 @@ const (
 	// The label is set when starting the Job, and removed on stopping the Job.
 	PodSetLabel = "kueue.x-k8s.io/podset"
 )
+
+// TopologySpec defines the desired state of Topology
+type TopologySpec struct {
+	// TopologyLevels define the levels of topology.
+	//
+	// +listType=atomic
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:MaxItems=5
```

Comment:
```suggestion
	// +kubebuilder:validation:MaxItems=5
	// +optional
```

Is this optional field?

Reply:
I don't think so, it only makes sense if you have at least one topology level.